src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/BulkUpload/BulkUploadValidator.cs

```diff
@@ -33,16 +32,12 @@ public sealed class BulkUploadValidator : IBulkUploadValidator
        
         public BulkUploadValidator(ProviderApprenticeshipsServiceConfiguration config, ILog logger, IUlnValidator ulnValidator, IAcademicYearDateProvider academicYear)
         {
-            if (logger == null)
-                throw new ArgumentNullException(nameof(logger));
             if (config == null)
```

Comment:
You could have removed the config == null check too, and the logger doesn't seem to be used.